projects/plugins/boost/app/features/guide/src/ui/Bubble.svelte

```diff
@@ -15,41 +16,77 @@
 		y: 2,
 		easing: backOut,
 	};
+
+	$: overiszedLabel = oversizedBy < 9 ? oversizedBy.toFixed(1) : `${Math.floor( oversizedBy )}+`;
```

Comment:
A small typo
```suggestion
	$: oversizedLabel = oversizedBy < 9 ? oversizedBy.toFixed(1) : `${Math.floor( oversizedBy )}+`;
```

Reply:
huh, weird that my editor thinks that `overiszed` is a word 🤷